lib/lib_rss.php

```diff
@@ -416,7 +416,7 @@ function check_install_php() {
 	$pdo_pgsql = extension_loaded('pdo_pgsql');
 	$pdo_sqlite = extension_loaded('pdo_sqlite');
 	return array(
-		'php' => version_compare(PHP_VERSION, '5.5.0') >= 0,
+		'php' => version_compare(PHP_VERSION, FRESHRSS_MIN_PHP_VERSION) >= 0,
```

Comment:
Would this check be obsolete too? Hard to tell on phone.

Reply:
I think the check has to be kept but the version it was tested against was. I think that was forgotten when changing the requirements. That's the reason why I want to have only have one constant value. Next time we need to change it, we won't face the same issue.